cli/pcluster/utils.py

```diff
@@ -136,7 +148,7 @@ def delete_s3_bucket(bucket_name):
     except boto3.client("s3").exceptions.NoSuchBucket:
         pass
     except ClientError:
-        print("Failed to delete bucket %s. Please delete it manually." % bucket_name)
+        LOGGER.critical("Failed to delete bucket %s. Please delete it manually." % bucket_name)
```

Comment:
What about setting this as a warning and also printing the reason why the deletion failed?

Reply:
done